extensions/amp-viewer-integration/0.1/test/test-highlight-handler.js

```diff
@@ -456,15 +456,49 @@ describes.realWin(
         );
       });
 
-    // TODO(dmanek): remove `ifChrome` once other major browsers support
-    // text fragments (i.e. 'fragmentDirective' in document = true)
+    // TODO(dmanek): remove `ifChrome` once other major browsers support text fragments
+    it.configure()
+      .ifChrome()
+      .run(
+        'should not highlight using text fragments for Chrome 92',
+        async () => {
+          const {ampdoc} = env;
+          const platform = Services.platformFor(ampdoc.win);
+          env.sandbox.stub(platform, 'isChrome').returns(true);
+          env.sandbox.stub(platform, 'getMajorVersion').returns(92);
+          let whenFirstVisiblePromiseResolve;
+          const whenFirstVisiblePromise = new Promise((resolve) => {
+            whenFirstVisiblePromiseResolve = resolve;
+          });
+          env.sandbox
+            .stub(ampdoc, 'whenFirstVisible')
+            .returns(whenFirstVisiblePromise);
+
+          const highlightHandler = new HighlightHandler(ampdoc, {
+            sentences: ['amp', 'highlight'],
+          });
+
+          const updateUrlWithTextFragmentSpy = env.sandbox.spy();
+          highlightHandler.updateUrlWithTextFragment_ =
+            updateUrlWithTextFragmentSpy;
+
+          whenFirstVisiblePromiseResolve();
+          await whenFirstVisiblePromise;
+
+          expect(updateUrlWithTextFragmentSpy).not.to.be.called;
+        }
+      );
+
+    // TODO(dmanek): remove `ifChrome` once other major browsers support text fragments
```

Comment:
If we retain the feature detection, would we need to run this test against Chromium only?

Reply:
Correct. If we were not doing Chrome version detection, this could be removed. I've updated the TODO.